Runtimes/Supplemental/CMakeLists.txt

```
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.29)
+
+project(SwiftRuntime LANGUAGES Swift C CXX)
+
+include(ExternalProject)
+
+set(SwiftRuntime_SWIFTC_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../../")
+
+foreach(lib ${Swift_ENABLE_RUNTIMES})
+  string(TOLOWER ${lib} name)
+  set(SwiftRuntime_ENABLE_${name} YES)
+endforeach()
+
+if(SwiftCore_DIR)
+  set(SwiftCore_DIR_FLAG "-DSwiftCore_DIR=${SwiftCore_DIR}")
```

Should we use `SwiftCore_ROOT` instead? If I'm following the [documentation](https://cmake.org/cmake/help/latest/command/find_package.html) properly, `<PackageName>_DIR` is supposed to be an env variable from an user perspective, and the matching cache variable is mostly managed by CMake itself (which can change it if invalid)

```suggestion
if(SwiftCore_ROOT)
  set(SwiftCore_ROOT_FLAG "-DSwiftCore_ROOT=${SwiftCore_ROOT}")
```